extensions/image/internal.m

```diff
@@ -778,6 +779,100 @@ static int imageFromApp(lua_State *L) {
     return 1;
 }
 
+/// hs.image.imageFromMediaFile(file) -> object
+/// Constructor
+/// Creates an `hs.image` object from a video file or the album artwork of an audio file or directory
+///
+/// Parameters:
+///  * file - A string containing the path to an audio or video file or an album directory
+///
+/// Returns:
+///  * An `hs.image` object or `nil`, if no album art was found
```

Comment:
"album art was found" -> "image could be created"

(although I find myself wondering whether we should return a nil here, or get the OS to give us the filetype image as a fallback)